src/reducers/index.js

```diff
@@ -0,0 +1,28 @@
+import { ADD_TASK, DELETE_TASK, EDIT_TASK, LOAD_TASK } from '../actions';
+
+let taskId = 0
+
+const tasks = [
+{
+  id:1,
+  title: "Feed Dogs",
+  createdBy: "Dad",
+  assignedTo:"Kid",
+  Priority: "High"
+}
+];
+
+const taskReducer = (state = tasks, action) => {
```

Comment:
When you get to this call me or another instructor over to talk through the structure of your store. Want our store to be an object rather than an array.